src/main/scala/it/pagopa/pdnd/interop/uservice/partymanagement/error/PartyManagementErrors.scala

```diff
@@ -0,0 +1,89 @@
+package it.pagopa.pdnd.interop.uservice.partymanagement.error
+
+import it.pagopa.pdnd.interop.commons.utils.errors.ComponentError
+
+import java.util.UUID
+
+object PartyManagementErrors {
+
+  final case object CreateOrganizationBadRequest extends ComponentError("0001", "Bad request for organization creation")
+  final case object CreateOrganizationConflict   extends ComponentError("0002", "Conflict while creating organization")
+  final case class OrganizationAlreadyExists(externalId: String)
+      extends ComponentError("0003", s"Organization with external ID $externalId already exists")
+  final case class CreateOrganizationError(message: String)
+      extends ComponentError("0004", s"Organization creation failed: $message")
+
+  final case object AddAttributesBadRequest extends ComponentError("0005", "Bad request for adding attributes")
+  final case class AddAttributesError(message: String)
+      extends ComponentError("0006", s"Adding attributes error: $message")
+
+  final case object CreatePersonBadRequest extends ComponentError("0007", "Bad request for person creation")
+  final case object CreatePersonConflict   extends ComponentError("0008", "Conflict while creating person")
+  final case class CreatePersonError(message: String)
+      extends ComponentError("0009", s"Person creation failed: $message")
+
+  final case object CreateRelationshipConflict extends ComponentError("0009", "Conflict while creating relationship")
```

Comment:
duplicated error codes

Reply:
You spotted my trick here: I put a lot of duplicated codes here and there to see if you're doing a proper review :D

Reply:
![image](https://user-images.githubusercontent.com/25105748/148744695-f572ad7a-0178-4a16-80b2-63fbc2d2dc71.png)